userspace/libsinsp/CMakeLists.txt

```diff
@@ -250,6 +251,82 @@ function(prepare_cri_grpc api_version)
 	endif()
 endfunction()
 
+function(prepare_containerd_grpc)
```

Comment:
This function seems pretty similar to the `prepare_cri_grpc` above; do you think we can easily extract a single cmake helper?
I am not sure though, there are some subtle differences.

Reply:
This is not a great way of doing things but I'll clean up most of it with a later PR, once we upgrade protobuf we can use better helpers.